src/realm/tokenizer.hpp

```diff
@@ -36,7 +36,7 @@ class Tokenizer {
 
     std::string_view get_token()
```

Comment:
Probably this can be marked `const` ... Are you intentionally exposing a mutable view for the string?